cedar-policy-validator/src/err.rs

```diff
@@ -134,6 +134,23 @@ impl SchemaError {
     }
 }
 
+#[derive(Debug)]
```

Comment:
Not sure if it makes sense to replace it with `Either<EntityUID, Name>`.

Reply:
I prefer helpful variant names (in this case, `ActionContext` and `EntityTypeShape`) to the bland ones `Left` and `Right`, particularly in this case where they provide additional information (that the `EntityUID` is for an action, for instance)

Reply:
We could even go farther and name the `EntityUID` field of the `ActionContext` variant, eg, `action_uid`.